README.md

```diff
@@ -80,7 +80,21 @@ note: Flutter might ask you which target you'd like to run.
 
 Running 10101 for `web` target is currently unsupported.
 
-#### Regtest
+### `regtest`
+
+To run on `regtest` you have to change the default network (`testnet`) to `regtest` by setting the `NETWORK` environment variable.
+
+Example command for maker:
+
+`NETWORK=regtest make maker`
+
+Example command for taker:
+
+`NETWORK=regtest flutter run`
+
```

Comment:
```suggestion

You might want to consider adding the following line to your .bashrc/.zshrc:

`export NETWORK=regtest`

in order to always run on regtest.
```

Reply:
I think that heavily depends on your setup - I don't really like to make these things global :)